### ISSUE TITLE
App window position

### ISSUE THREAD

**Issue by rikublock** [GemWallet/gemwallet-extension#223]
Clicking the extension icon opens the app window (borderless, no menu) in place. Whereas using the API creates a popup-like app window (with border, menu) somewhere on the screen. (This behavior could be dependent on the OS)

Would be nice, if this was more consistent (especially the positioning). Ideally the icon app window is used for all cases.

Opened with icon (positioned next to icon):
![app_borderless](https://github.com/GemWallet/gemwallet-extension/assets/46352032/83018bb7-794e-44f8-a13a-f83b85aa42ee)

Opened with API call(positioned somewhere top-left of screen):
![app_border](https://github.com/GemWallet/gemwallet-extension/assets/46352032/d930658b-1d59-4509-afc1-429004e395f1)

**Comment by FlorianBouron** [comment]
Hi @rikublock ,
Thank you for the feedback, do you mean that you would put the window on the top right instead?
I see that it is actually what does Phantom,

**Comment by rikublock** [comment]
> Thank you for the feedback, do you mean that you would put the window on the top right instead?

Yeah pretty much. I would try to open the exact same window (same position) as if the user clicked the extension icon. Not sure what is happening behind the scene currently, but sort of looks like two different things are being used (popup window for API and extension window/menu for icon click).

> I see that it is actually what does Phantom,

I think pretty much any extension does it. Metamask would be another (good) example. 

It's probably not very high priority, but it's certainly annoying, if the window keeps jumping around the screen.
